sdk/.gitignore

```diff
@@ -1 +1,8 @@
-workspace-packages.json
\ No newline at end of file
+workspace-packages.json
+
+*.js
```

Comment:
i don't understand why all js and ts files are to be ignored under sdk. in this review we'd have rollup.config.js file under sdk folder, is that no longer needed?